packages/app/src/app/components/CodeEditor/Configuration/index.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { TextOperation } from 'ot';
 import { Module } from '@codesandbox/common/lib/types';
 import getUI from '@codesandbox/common/lib/templates/configuration/ui';
-import getType from 'app/utils/get-type';
+import getType from 'app/utils/get-type.ts';
```

Comment:
This one doesn't need an extension methinks, since we're in a TS file

Reply:
Updated

Reply:
Don't work